spec/features/shared/footer_spec.rb

```diff
@@ -27,6 +27,9 @@
         expect(page).to have_link('Nominate a practice')
       end
     end
+    it 'should open feedback modal' do
+      find('#feedback-modal').click
+    end
```

Comment:
Just one more thing here Brad. For this particular test, we should be expecting some sort of outcome. Can we add an expectation for the `#feedback-modal` element?

Reply:
oops.. added assertion - I think I removed it.. bc the javascript wasn't being invoked with the link click.. but with fix put in yesterday - modal pops up in rspec no problem.  thx Josh